packages/pharos/src/components/toast/pharos-toaster.ts

```diff
@@ -6,6 +6,8 @@ import { toasterStyles } from './pharos-toaster.css';
 import type { PharosToast } from './pharos-toast';
 import { DEFAULT_STATUS, DEFAULT_ID, DEFAULT_INDEFINITE } from './pharos-toast';
 
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
You'll want to add `uuid` as a [direct dependency for the package](https://github.com/ithaka/pharos/blob/develop/packages/pharos/package.json#L56) to ensure consumers get it installed.

Reply:
added!